charts/opentelemetry-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.7.0
+version: 0.7.1
```

Comment:
```suggestion
version: 0.8.0
```

Reply:
Most of the changes were white space, there was only one additive change.